diff -urN aa/gpdf-0.122/xpdf/gpdf-font-face.c gpdf-0.122/xpdf/gpdf-font-face.c
--- aa/gpdf-0.122/xpdf/gpdf-font-face.c	2003-06-28 19:39:00.000000000 +0200
+++ gpdf-0.122/xpdf/gpdf-font-face.c	2004-02-06 20:34:47.434322296 +0100
@@ -151,7 +151,6 @@
 	entry->speciesname = g_strdup (species
 				       ? species
 				       : (const guchar*) "Regular");
-	entry->weight = entry->speciesname; /* FIXME here and in fcpattern_to_gp_font_entry */
 	entry->Weight = weight;
 	entry->familyname = g_strdup (family);
 	entry->name = g_strconcat (family, " ", entry->speciesname, NULL);
